### PR TITLE
fix: pagination consistent width across html and react platforms

### PR DESCRIPTION
**Pull request by hamza14khan**
- Pagination (react and html): usage icon classes on next and previous buttons if no label is shown (e.g. under 640px - small screens).
- Extended class names on button macro.
- Documentation update.

Testing:
Screenshot 2024-11-25 at 22 40 31

### CODE REVIEW

File: packages/react/ds/src/pagination/pagination.tsx [hunk 1]
```diff
@@ -18,8 +18,9 @@ export const Pagination: React.FC<PaginationProps> = ({
   onPageChange,
 }) => {
   const { breakpoint, width } = useBreakpoint();
-  const isCompactView = breakpoint === Breakpoint.XS; // Custom breakpoint for compact view.
-  const showLabel = width < 639;
+  const isCompactView = breakpoint === Breakpoint.XS;
+  const isSMWidth = width < 639;
+  const showLabel = isSMWidth;
```
Comment (pregno) [inline packages/react/ds/src/pagination/pagination.tsx:23]: If `showLabel` and `isSMWidth` are the same, why having 2 const?

File: apps/docs/dictionary.txt [hunk 2]
```diff
@@ -122,4 +122,8 @@ itemsDistribution
 fixedHeight
 groupId
 hideToc
-Dismissible
\ No newline at end of file
+Dismissible
+gi-pagination-next-btn
```
Comment (hamza14khan) [inline apps/docs/dictionary.txt:126]: Do you guys know why do I have to add classes in dicitonary.txt? 🤔 

@pregno @ramonmulia @BogdanDanilescu

Comment (pregno) [reply apps/docs/dictionary.txt:126]: Because we are adding the `html` in the mdx file.

Comment (hamza14khan) [reply apps/docs/dictionary.txt:126]: It never complained on other HTML files using classes?

**Review by ramonmulia — APPROVED** [review]

File: packages/design/tailwind/css/components.css [hunk 3]
```diff
@@ -904,10 +904,21 @@ input[type='file' i] {
...
 /* End of Toast */
+
 /* Pagination */
 
 .gi-pagination {
-  @apply gi-flex xs:gi-justify-center gi-justify-between gi-items-center  gi-gap-2;
+  @apply gi-flex xs:gi-justify-center gi-justify-between gi-items-center gi-gap-2;
 }
 
+@media (max-width: 640px) {
```
[the remark this answers is not in the document]
Comment (hamza14khan) [reply packages/design/tailwind/css/components.css:914]: It throws me this error:

The `max-sm:gi-p-3` class does not exist. If `max-sm:gi-p-3` is a custom class, make sure it is defined within a `@layer` directive.

When using this:

```
.gi-pagination-next-btn,
.gi-pagination-prev-btn {
  @apply max-sm:gi-p-3 max-sm:gi-h-12 max-sm:gi-w-12 max-sm:gi-justify-center;
}
```

File: apps/docs/dictionary.txt (hunk 2 above)
Comment (pregno) [reply apps/docs/dictionary.txt:126]: It's checking only mdx files

**Review by pregno — APPROVED** [review]

**Pull request merged**